api/endpoints/members.py

```diff
@@ -691,13 +691,17 @@ def get(self):
                     "Sid": "GrantAccessToUserFolder",
                     "Effect": "Allow",
                     "Action": [
-                        "s3:ListBucket",
-                        "s3:DeleteObject",
-                        "s3:GetObject",
-                        "s3:PutObject",
-                        "s3:RestoreObject",
-                        "s3:ListMultipartUploadParts",
-                        "s3:AbortMultipartUpload"
+                        "s3:*"
```

Comment:
Is there a reason to use `s3:*` here ? And not explicitly specify the permissions ?